### PR TITLE
new pkg: aws-cli (v2), rename aws-cli to aws-cli-v1

### PR DESCRIPTION
**Pull request by tuananh**
Tested by creating an apko image

image

I tried moving documentation to sub-package but apparently, they use it for `aws help` command. so we will need to ignore melange documentation linter as well.

Fixes:

Related: #1848 #2272 #3884

#### For new package PRs only

- [x] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [x] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: aws-cli.yaml [hunk 1]
```diff
@@ -1,45 +1,52 @@
...
   contents:
     packages:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - py3-setuptools
-      - python3
-      - python3-dev
+      - posix-libc-utils # need ldd
+      - python-3.11
```
Comment (dungdm93) [inline aws-cli.yaml:21]: Can we use virtual package instead?

**Comment by ajayk** [comment]
@tuananh will pick this up this week , sorry for the delay though

**Comment by ajayk** [comment]
Hi @tuananh can you please add the new package under aws-cli-v2 , leave the existing package as it is to avoid any confusions

**Comment by tuananh** [comment]
@ajayk updated as your recommendation as well as pumping up to latest version.

verify it works with `LD_LIBRARY_PATH: /usr/lib/aws-cli`. related issue: https://github.com/aws/aws-cli/issues/5506

Not sure how to best handle this.

**Comment by Dentrax** [comment]
Thanks for working on this @tuananh!

According to your last comment, can we able to tackle `LD_LIBRARY_PATH` issue using `scriptlets`?

```yaml
scriptlets:
    post-install: |
      #!/bin/sh
      echo
      echo "WARNING: You need to run 'export LD_LIBRARY_PATH=/usr/lib/aws-cli' to use this package."
      echo
```

We can also add this in the image manifest for sure.

**Review by ajayk — DISMISSED** [review]

**Comment by tuananh** [comment]
Not sure how this got in

```
Building package pgaudit-14 with version pgaudit-14-1.6.2-r1 from file ./pgaudit-14.yaml
make yamlfile=./pgaudit-14.yaml srcdirflag="--source-dir ./pgaudit-14" pkgname=pgaudit-14 packages/x86_64/pgaudit-14-1.6.2-r1.apk
```

**Review by ajayk — APPROVED** [review]

**Pull request merged**